File: replace.py
import re

with open('index.html', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('placeholder="הכנס שם כאן..."', 'placeholder="הכנס שם כאן... (מומלץ עם אימוג\'י)"')

with open('index.html', 'w', encoding='utf-8') as f:
    f.write(content)

print("Replaced successfully")

File: replace.js
const fs = require('fs');
let content = fs.readFileSync('index.html', 'utf8');
content = content.replace(/<div style="text-align: right; margin-bottom: 25px;">.*?<input type="hidden" id="custom-emoji" value="[^"]+">\s*<\/div>/s, '<input type="hidden" id="custom-emoji" value="🎯">');
fs.writeFileSync('index.html', content, 'utf8');
